examples/label_studio_annotation/README.md

```diff
@@ -0,0 +1,318 @@
+
+This example requires a few more steps than usual, given that it requires a
+cloud artifact store. Instructions for Azure, AWS S3 and GCP can be found in the
+following guide.
+
+## 🗺 Overview
+
+NOTE: This example currently only runs with a cloud artifact store. [This guide
+by Label Studio](https://labelstud.io/guide/storage.html) contains more details
+if you have trouble understanding the instructions below.
+
+For a full video walkthrough on how to run this email, please check out [our
+community hour demo](https://www.youtube.com/watch?v=bLFGnoApWeU) in which we
+show how to run the various pipelines, in what order, all using an AWS stack.
+
+# 🖥 Run the example
+
+## 👣 Step-by-Step
+
+### 🥞 Set up your stack for Microsoft Azure
+
+In order to run this example, you need to install and initialize ZenML and Label
+Studio.
+
+```shell
+zenml integration install label_studio pytorch azure
+
+# pull example
+zenml example pull label_studio_annotation
+cd zenml_examples/label_studio_annotation
+
+# Initialize ZenML repo
+zenml init
+```
+
+Some setup for your stack is required. This assumes you have a cloud secrets
+manager setup and installed. (See [here](https://docs.zenml.io/mlops-stacks/secrets-managers) for more on how to do that.)
+
+```shell
+zenml stack copy default <ANNOTATION_STACK_NAME>
+
+zenml stack set <ANNOTATION_STACK_NAME>
+
+zenml secrets-manager register <YOUR_SECRETS_MANAGER> --key_vault_name=<YOUR_KEY_VAULT_NAME> -f azure_key_vault
+
+zenml stack update <ANNOTATION_STACK_NAME> -x <YOUR_SECRETS_MANAGER>
+
+zenml secret register <YOUR_AZURE_AUTH_SECRET_NAME> --schema=azure --account_name="<YOUR_AZURE_ACCOUNT_NAME>" --account_key="<YOUR_AZURE_ACCOUNT_KEY>"
+
+zenml artifact-store register azure_artifact_store -f=azure --path="az://<NAME_OF_ARTIFACT_STORE_OR_BLOB_IN_AZURE>" --authentication_secret="<YOUR_AZURE_AUTH_SECRET_NAME>"
+
+zenml stack update <ANNOTATION_STACK_NAME> -a <YOUR_CLOUD_ARTIFACT_STORE>
+```
+
+You will next need to obtain your Label Studio API key. This will give you access to the web annotation interface.
+
+```shell
+# start a temporary / one-off label-studio instance to get your API key
+label-studio start -p 8094
+```
+
+First order of business is to sign up if you haven't done so already, using an email and a password. Then visit [http://localhost:8094/](http://localhost:8094/) to log in, and then visit [http://localhost:8094/user/account](http://localhost:8094/user/account) and get your Label Studio API key (from the upper right hand corner). You will need it for the next step. `Ctrl-c` out of the Label Studio server that is running on the terminal.
+
+```shell
+zenml secret register <LABEL_STUDIO_SECRET_NAME> --api_key="<YOUR_API_KEY>"
+
+zenml annotator register <YOUR_LABEL_STUDIO_ANNOTATOR> --flavor label_studio --authentication_secret="<LABEL_STUDIO_SECRET_NAME>"
+
+zenml stack update <ANNOTATION_STACK_NAME> -an <YOUR_LABEL_STUDIO_ANNOTATOR>
+
+zenml stack up
+```
```

Comment:
This is also not cloud provider-specific. Similar to above, I would suggest to move this to a separate section "Label Studio Setup" or similar instead of having it duplicated 3x.

Reply:
Also has been moved, but the stack setup steps can't be moved elsewhere unfortunately as there are some order dependencies that are a bit fiddly.